docs/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,9 @@
+### Description
```

Comment:
I think file name should be renamed to `pull_request_template.md`, by making template Markdown, in future, we can use features like Markdown formatting, @-mentions, and task lists